packages/example-app-base/src/utilities/getPathMinusLastHash.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Retrieve the route URL for a page without the string after the last hash.
+ */
+export function getPathMinusLastHash(path: string): string {
```

Comment:
nit: naming (path is explicitly the bit before the hash) https://en.wikipedia.org/wiki/URL#/media/File:URI_syntax_diagram.png

Reply:
What would you recommend instead? getUrlMinusLastHash? Also, the URLs in this context can be a bit strange in that they can potentially have two hashes--the first for the route and the second for the anchor within the page.